livekit-ffi/protocol/room.proto

```diff
@@ -128,6 +128,18 @@ message UpdateLocalMetadataCallback {
   uint64 async_id = 1;
 }
 
+// Change the local participant's attributes
+message UpdateLocalAttributesRequest {
```

Comment:
nit: UpdateLocalAttributesRequest vs UpdateAttributesRequest?

Reply:
kept consistency here with others (`UpdateLocalMetadataRequest`)

Since i'll changing Update -> Set in this PR, want me to get rid of Local?